lib/use-cases/UpdateArchiveStatus.test.js

```diff
@@ -17,16 +17,11 @@ describe('use-case/updateArchiveStatus', () => {
   const updateArchiveStatus = require('./UpdateArchiveStatus')(options);
 
   describe('with an invalid dropbox id', () => {
-    beforeEach(() => {
-      options.gateways.dropbox.get.mockImplementation(() => {
-        return Promise.resolve(null);
-      });
-    });
+    options.gateways.dropbox.get.mockImplementation(() => null);
 
     it('throws an "invalid dropbox" error', async () => {
```

Comment:
The name of this test no longer fits with the assertions, probably worth renaming since this doesn't throw an error any more?